Source/settings/mods.js

```diff
@@ -39,11 +39,11 @@ var FullScreenPokemon;
                 events: {
                     onModEnable: function (mod) {
                         var stats = this.ObjectMaker.getFunction("Player").prototype;
-                        this.player.speed = stats.speed = 10;
+                        stats.speed = 10;
                     },
                     onModDisable: function (mod) {
                         var stats = this.ObjectMaker.getFunction("Player").prototype;
-                        this.player.speed = stats.speed = this.settings.objects.properties.Player.speed;
```

Comment:
This was added when I did a `git pull` from FSS mods. Not sure why it was included in the PR